packages/ngx-forms/src/lib/datepicker/components/datepicker/datepicker.component.ts

```diff
@@ -123,11 +123,11 @@ export class DatepickerComponent implements OnInit, OnDestroy, ControlValueAcces
       .build();
   }
 
-  public writeValue(value: string): void {
-    const date = DateHelper.parseDate(value, 'DD/MM/YYYY', true);
-    const dateString = date ? this.formatDate(date) : '';
-
-    this.selectedDate = date;
+  public writeValue(value: string | Date): void {
+    this.selectedDate = typeof value === 'string'
```

Comment:
isn't it safer to move the typeof check inside the isISODateFormat function? That way it can't be used in an improper way.

Reply:
The check is placed here because we accept both Date and String values.
I added the string type to the function parameter so now the "isISODateFormat" function only accepts strings.